src/styles/_catalog.scss

```diff
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  gap: 48px;
   margin: 0 auto;
 
```

Comment:
```suggestion  

```

Reply:
Looks like it's not applied anymore, as there is no width. The work is done with justify-content: center :)